upgrade/upgrade.go

```diff
@@ -73,6 +73,7 @@ func Upgrade(kubeconfigPath, currentNodeID string) error {
 
 func upgrade(currentNodeID, namespace string, config *restclient.Config, lhClient *lhclientset.Clientset, kubeClient *clientset.Clientset) error {
 	ctx, cancel := context.WithCancel(context.Background())
+	errPointer := new(error)
```

Comment:
Can we define `err` here? Instead of defining it again in the function.

Reply:
Sorry. My bad. I didn't really understand variable scope here. I was thinking doing like that will create 2 copies of err variable. But it turn out it does not. It create one variable in heap. 

Thanks, fixed